MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,51 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace MagneticFieldParabolicPortable {
+
+    struct Parameters {
+      // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
+      // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+      static constexpr float c1 = 3.8114;
+      static constexpr float b0 = -3.94991e-06;
+      static constexpr float b1 = 7.53701e-06;
+      static constexpr float a = 2.43878e-11;
+      static constexpr float max_radius2 = 13225.f;  // tracker radius
+      static constexpr float max_z = 280.f;          // tracker z
+    };
+
+    template <typename Vec3>
+    constexpr float Kr(Vec3 const& vec) {
+      return Parameters::a * (vec(0) * vec(0) + vec(1) * vec(1)) + 1.;
+    }
+
+    template <typename Vec3>
+    constexpr float B0Z(Vec3 const& vec) {
+      return Parameters::b0 * vec(2) * vec(2) + Parameters::b1 * vec(2) + Parameters::c1;
+    }
+
+    template <typename Vec3>
+    constexpr bool isDefined(Vec3 const& vec) {
+      return ((vec(0) * vec(0) + vec(1) * vec(1)) < Parameters::max_radius2 && fabs(vec(2) < Parameters::max_z));
+    }
+
+    template <typename Vec3>
+    constexpr float MagneticFieldAtPoint(Vec3 const& vec) {
```

Comment:
functions should be lower case, so `magneticFieldAtPoint()` or `getMagneticFieldAtPoint()`

Reply:
OK: https://github.com/cms-sw/cmssw/pull/44360/commits/d005bcc5758caa07bf26ced6dc464ed3f134be2c